Testing-Notes.md

```diff
@@ -0,0 +1,42 @@
+# Testing Notes
+
+## `/login`
+- `"username"` should not be case sensitive. It should be normalised before it is added to the db. <br>
+**Example**: passing `{"username": "Pavol"}` should be normalised to `{"username": "pavol"}`.
```

Comment:
[Trello card](https://trello.com/c/qUxjuugg/67-normalize-username-before-saving-it-to-db)